versioning_function.sql

```diff
@@ -148,7 +148,12 @@ BEGIN
       AND history.attname != sys_period;
 
     EXECUTE ('INSERT INTO ' ||
-      quote_ident(history_table) ||
+      CASE split_part(history_table, '.', 2)
```

Comment:
Using a fully qualified name as the history table should be supported - it is in the extension version.

Reply:
This was necessary because of this error (simplified):

```
ERROR:  relation "schema_name.table_history" does not exist at character 13
QUERY:  INSERT INTO "schema_name.table_history"(table_id,valid_between) VALUES ($1.table_id,tstzrange($2, $3, '[)'))
CONTEXT:  PL/pgSQL function versioning() line 149 at EXECUTE
```